JumpscaleCore/core/BASECLASSES/JSConfigsBCDB.py

```diff
@@ -83,19 +83,21 @@ def _new(self, name, jsxobject=None, save=True, **kwargs):
                 jsxobject = self._model.new()
             jsxobject.name = name
 
-        jsxobject._autosave = True
+        jsxobject._autosave = save
 
         # means we need to remember the parent id
-        mother_id = self._mother_id_get()
+        mother_id, has_mother = self._mother_id_get()
+        if has_mother and not mother_id and save:
+            raise j.exceptions.Value("Save can't be True if parent is not saved")
+
         if mother_id:
             if jsxobject.mother_id != mother_id:
                 jsxobject.mother_id = mother_id
-                jsxobject.save()
```

Comment:
Again, we can't just save an object without respecting the value of `save`. Besides, this is completely pointless because at the end of the function the object is saved.